src/core/server/http/security_response_headers_config.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { schema, TypeOf } from '@kbn/config-schema';
+
+export const securityResponseHeadersSchema = schema.object({
+  strictTransportSecurity: schema.oneOf([schema.string(), schema.literal(null)], {
+    // See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
+    defaultValue: 'max-age=31536000', // 1 year in seconds
```

Comment:
How does `Strict-Transport-Security: max-age=31536000` affect Kibana when it's hosted on a base-path? For example, if Kibana is hosted at `https://foo.com/kibana/` will it apply to websites at `https://foo.com/bar`? If it's in effect for the entire domain, at a minimum, we won't want to set this in 7.13.

Reply:
http://gph.is/1lX4Z3B

I'll make an issue to change the defaults for `strictTransportSecurity` and `disableEmbedding` in the 8.0 release.